.github/workflows/ci.yml

```diff
@@ -65,8 +65,8 @@ jobs:
     strategy:
       matrix:
         python:
-        - "3.8"  # oldest Python supported by PSF
-        - "3.12"  # newest Python that is stable
+        - "3.8"   # oldest Python supported by validate-pyproject
+        - "3.x"   # newest Python that is stable
```

Comment:
@abravalheri, personally, I prefer listing this explicitly (`3.13`), then updating it as soon as a beta comes out (leaving `allow-prereleases: true`). But that's also because I list the Python versions in trove classifiers, and once it passes for the first beta is the point I declare support for the new version.

Reply:
Thank you very much @henryiii for assessing this.
I tend to simplify as much as I can the classifiers (I am not using them for the time being to record which versions the project is tested).

In the future we can change that if required.

Also open to have `allow-prereleases: true` (as long as we don't require the pre-releases to pass). But we probably can have that in a separated PR if there is anyone in the community interested in that.

For the time being the GHA CI is kept very lightweight the bulk of testing happens on cirrus (also open to review that if required).